src/System.Console/src/System/ConsolePal.Windows.cs

```diff
@@ -14,11 +14,9 @@ internal static class ConsolePal
     {
         private const int DefaultConsoleBufferSize = 256; // default size of buffer used in stream readers/writers
 
-        private static IntPtr s_InvalidHandleValue = new IntPtr(-1);
+        private static IntPtr InvalidHandleValue => new IntPtr(-1);
```

Comment:
So worth keeping this one?

Reply:
This one was used in a number of places so I have changed it to property that will get inlined. It is no longer changed in a static.